gov_uk_dashboards/components/plotly/tooltip.py

```diff
@@ -0,0 +1,21 @@
+"""tooltip"""
+from dash import html
+import dash_bootstrap_components as dbc
+
+
+def tooltip(tooltip_text, tooltip_id):
+    """Creates a tooltip for providing further information"""
```

Comment:
I'd recommend adding some typehints and better documentation of the two arguments.  I find the `tooltip_id` very mysterious.

[table_from_dataframe](https://github.com/communitiesuk/pkg_gov_uk_dashboards/blob/main/gov_uk_dashboards/components/plotly/table.py#L7-L36) is a good example.